### ISSUE TITLE
Exception on quote in actor name

### ISSUE THREAD

**Issue by jellyfish77** [jellyfish77/camel-eip#4]
```
ax.xml.transform.TransformerException: misquoted literal... expected single quote!
	at com.sun.org.apache.xpath.internal.compiler.XPathParser.error(XPathParser.java:612)
	at com.sun.org.apache.xpath.internal.compiler.Lexer.tokenize(Lexer.java:203)
	at com.sun.org.apache.xpath.internal.compiler.Lexer.tokenize(Lexer.java:100)
	at com.sun.org.apache.xpath.internal.compiler.XPathParser.initXPath(XPathParser.java:114)
	at com.sun.org.apache.xpath.internal.XPath.<init>(XPath.java:180)
	at com.sun.org.apache.xpath.internal.XPath.<init>(XPath.java:268)
	at com.sun.org.apache.xpath.internal.jaxp.XPathImpl.compile(XPathImpl.java:390)
	at movies.OmdbServiceBean.enrichMovie(OmdbServiceBean.java:185)
```

**Comment by jellyfish77** [comment]
Reproduce using 'Jurassic World'

**Comment by jellyfish77** [comment]
Fixed 60a3edf
